packages/server/lib/controllers/proxy.controller.ts

```diff
@@ -92,7 +93,8 @@ class ProxyController {
                 headers,
                 baseUrlOverride,
                 decompress: decompress === 'true' ? true : false,
-                method: method.toUpperCase() as HTTP_VERB
+                method: method.toUpperCase() as HTTP_VERB,
+                retryOn: retryOn && Array.isArray(retryOn) ? retryOn.map(Number) : null
```

Comment:
it's already mapped as Number so I think it can be as simple as that, no?
```suggestion
                retryOn,
```

Reply:
Yup, thanks. Updated with https://github.com/NangoHQ/nango/pull/1742/commits/8377b14dcf7555fdc788a75f11d9ceea4f6ce8c5 in addition to documentation